.github/workflows/prettier.yml

```diff
@@ -0,0 +1,24 @@
+name: Prettier Markdown
+
+on:
+  push:
+    branches: [master]
+    paths:
+      - 'docs/**'
```

Comment:
As mentioned we'd also need to turn off prettier checks in the CI build so that people can commit without issues, otherwise this workflow won't ever have anything to do

Reply:
I will change this to be on pull request